conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -48,8 +48,19 @@ def __init__(self, conanfile, namespace=None):
         self._target = None
 
         self.apple_arch_flag = self.apple_isysroot_flag = None
-
         self.apple_min_version_flag = apple_min_version_flag(self._conanfile)
+
+        self.msvc_runtime_flag = None
+        if is_msvc(self._conanfile):
```

Comment:
This will only work for ``compiler=msvc`` but not for ``compiler=Visual Studio`` as they have different sub-settings. I am fine with it, but ConanCenter still uses ``Visual Studio``, the migration to ``msvc`` will take some time.

Reply:
ok, will fix it.

Reply:
Review again please